packages/styles/src/core/layout/toolbar/_main.scss

```diff
@@ -1,36 +1,39 @@
+$_toolbar-main__size: 13 * $u1;
+
 .as-toolbar-main {
   display: flex;
-  height: 52px;
-  background-color: var(--brand-02-color, $brand-02-color);
-
-  &--bg-brand-1 {
-    background-color: var(--brand-01-color, $brand-01-color);
-  }
-
-  &--bg-brand-2 {
-    background-color: var(--brand-02-color, $brand-02-color);
-  }
+  height: $_toolbar-main__size;
+  background-color: var(--toolbar-main-bg-color, $brand-02-color);
 
-  &--bg-brand-3 {
-    background-color: var(--brand-03-color, $brand-03-color);
+  &__toggle {
+    position: absolute;
+    z-index: 1;
   }
 
   &__item {
     display: flex;
     align-items: center;
+    justify-content: center;
```

Comment:
Let's keep it as I did in my PR, we don't need to force users to use a fixed width/height